tests/mail/tests.py

```diff
@@ -422,6 +422,33 @@ def test_attach_text_as_bytes(self):
         self.assertEqual(content, b'file content')
         self.assertEqual(mimetype, 'text/plain')
 
+    def test_attach_utf8_text_as_bytes(self):
+        """
+        Make sure that non-ASCII characters encoded as valid UTF-8 get correctly
```

Comment:
chop "Make sure that" prefix. e.g. "Non-ASCII characters encoded as valid UTF-8 is correctly transported and decoded." (better yet, explain what "correctly" means)